packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -373,7 +416,55 @@ describe('connectInfiniteHits', () => {
       expect(props.hasMore).toBe(false);
     });
 
-    it('adds 1 to page when calling refine', () => {
+    it('calls refine with next page when calling refineNext', () => {
+      const context = createSingleIndexContext();
+      const getProvidedProps = connect.getProvidedProps.bind(context);
+
+      const hits = [{}, {}];
+
+      const props = getProvidedProps(
+        {},
+        {},
+        {
+          results: {
+            hits,
+            page: 2,
+            hitsPerPage: 2,
+            nbPages: 3,
+          },
+        }
+      );
+
+      props.refineNext.apply(context);
+
+      expect(context.refine.mock.calls[0][0]).toEqual(3);
+    });
+
+    it('calls refine with previous page when calling refinePrevious', () => {
+      const context = createSingleIndexContext();
+      const getProvidedProps = connect.getProvidedProps.bind(context);
+
+      const hits = [{}, {}];
+
+      const props = getProvidedProps(
+        {},
+        {},
+        {
+          results: {
+            hits,
+            page: 2,
+            hitsPerPage: 2,
+            nbPages: 3,
+          },
+        }
+      );
+
+      props.refinePrevious.apply(context);
+
+      expect(context.refine.mock.calls[0][0]).toEqual(1);
```

Comment:
We could use [`toHaveBeenCalledTimes`](https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber) + [`toHaveBeenLastCalledWith`](https://jestjs.io/docs/en/expect#tohavebeenlastcalledwitharg1-arg2-) to have a clearer intent.